app/src/screens/teams/TeamManageScreen.tsx

```diff
@@ -17,9 +22,22 @@ type TeamManageScreenProps = object &
 export const TeamManageScreen: React.FC<TeamManageScreenProps> = ({
   navigation,
 }) => {
+  const userInfo = useUserProfileQuery();
```

Comment:
Skoro hook zwraca profil użytkownika to może warto nazwać zmienną `userProfile`, będzie można łatwo skojarzyć z typem.